utils/sv_easing_functions.py

```diff
@@ -328,35 +328,46 @@ def BounceEaseInOut(p, settings=defaultBounceSettings):
 
 
 easing_dict = {
+    # LINEAR
```

Comment:
Make your own mapping dict, but please leave the existing dict unchanged (for said reasons legacy reasons). Either literally make a copy and rename it.. and restore the old one, or ( don't do this..) make a mapping-mapping dict. (don't do this)